Automattic/jetpack
# Update/improve installing state for recommended features

This PR updates the UX around installing/ enabling recommended features

#### Changes proposed in this Pull Request:

* This PR moves the state management for determining if a recommended feature is "installing" into the redux state so that the installing indicator shows the intermediate "waiting" state if the summary screen is reached before the feature is fully active/ installed
* Features that are in progress of being installed/ activated are now shown in the "Recommendations Enabled" section on the summary screen instead of the "Recommendations Skipped" section.

#### Other information:


#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Navigate to the Jetpack dashboard and click on the "Recommendations" navigation item
* Click through the recommendations flow, skipping all recommendations to arrive at the summary screen.
* Click to enable or install one of the recommended features. While enabling/ installing, the feature should move to the "Enabled Recommendations" section
![Screen Shot 2022-08-12 at 11 22 10 AM](https://user-images.githubusercontent.com/18016357/184390852-5af55a6c-efc8-4208-a282-40224ae0f683.png)

* Disable all features/ deactivate the plugins that you just activated.
* To see the state continuity between the individual steps and the summary screen, click on the name of one of the recommendations on the summary screen. From the individual step, click on the CTA to install or enable the feature. Quickly click the "View Summary" link on the next recommendation that shows to jump to the summary screen. You should see the feature that you just enabled shows in the installing state.
* This is easiest to see with the "Site Accelerator" recommendation, since you are directed to the summary screen immediately after enabling it.
